spec/helpers/items_helper_spec.rb

```diff
@@ -6,7 +6,7 @@
   describe '#stacks_url_full_size' do
     subject { helper.stacks_url_full_size('druid:999', 'foo/bar.txt') }
 
-    it { is_expected.to eq 'https://stacks-test.stanford.edu/file/druid:999/foo%2Fbar.txt' }
+    it { is_expected.to eq 'https://stacks-test.stanford.edu/file/druid:999/foo/bar.txt' }
```

Comment:
I can't explain the coverage dings given this is a whole-ass test for the helper.